index.html

```diff
@@ -45,7 +45,16 @@
         </div>
 
         <div class="control">
-            Material Properties
+            Materials
+        </div>
+
+        <div class="pill-nav" id="selectableMaterials">
```

Comment:
html provides a native select element. Would that meet your needs? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select